exercises/allergies/allergies_test.go

```diff
@@ -42,11 +42,11 @@ func TestAllergicTo(t *testing.T) {
 
 func BenchmarkAllergicTo(b *testing.B) {
 	b.StopTimer()
-	for _, test := range allergicToTests {
-		for _, response := range test.expected {
+	for i := 0; i < b.N; i++ {
+		for _, test := range allergicToTests {
 			b.StartTimer()
 
-			for i := 0; i < b.N; i++ {
+			for _, response := range test.expected {
 				AllergicTo(test.score, response.substance)
 			}
 			b.StopTimer()
```

Comment:
When benchmarking the AllergicTo, I've discovered that its best to omit all the b.StopTimer, b.StartTimer calls; otherwise the test runs for about two minutes on my system. The extra overhead of actually doing the timer manipulation made a huge difference with the loop index changes.  Did you also see that behavior?

Reply:
@leenipper yeah, same problem.